hosting/models.py

```diff
@@ -129,9 +132,41 @@ def is_in_book(self):
     def places_confirmed(self):
         return all(p.confirmed for p in self.owned_places.filter(deleted=False, in_book=True))
 
+    @property
+    def supervised_by(self):
+        places = self.owned_places.filter(deleted=False)
+        return [superv for p in places for superv in p.supervised_by]
+
+    @property
+    def is_supervisor(self):
+        return any(self.supervisor_of)
+
+    @property
+    def supervisor_of(self, code=False):
+        countries = (Country(g.name) for g in self.user.groups.all() if len(g.name) == 2)
+        return countries if code else [c.name for c in countries]
+
+    def is_supervisor_of(self, profile=None, countries=None):
+        """Compare intersection between responsabilities and given countries."""
+        countries = countries if countries else []
```

Comment:
> Haha, neee! Amatora pitonisto!

hmmm. veras. sed en tiu ĉi kazo tio ne influas! vi ja nur inspektas la valoron, konvertante ĝin al `Set`.